tuist/tuist
# Add --additional-parameters option to passthrough following arguments/options/flags to the `xcodebuild` invocation

Add the `--additional-parameters` option as a first step towards solving https://github.com/tuist/tuist/issues/5080.

### Short description 📝

Add a new `--additional-parameters` to `tuist test`, allowing to pass any arguments/options/flags to `xcodebuild`.

### How to test the changes locally 🧐

1. Run `tuist test --additional-parameters -dry-run` on any project and verify that the `xcodebuild` command actually does a dry run and doesn't do any testing.

### Contributor checklist ✅


### Reviewer checklist ✅
